datasets/hindiDiscourse/hindi_discourse.py

```diff
@@ -0,0 +1,91 @@
+# coding=utf-8
```

Comment:
can you move the directory from `hindiDiscourse` to `hindi_discourse` please, to match the dataset script name ?

Reply:
basically the directory name and the python file name should be the same right?

Reply:
yes exactly

Reply:
they should both be named `hindi_discourse`